### PR TITLE
fix(AddCard) added proper color to title

### LINKED ISSUE #474: [AddCard] Text color is wrong
image

Should be darker, matching the icon

### PR DESCRIPTION
**Pull request by sstone2423**
Closes #474 

**Summary**

- Made more descriptive class name for title
- Added `$text-01` to the title's class to match icon

**Before**
Screen Shot 2020-01-28 at 10 50 21 PM

**After**
Screen Shot 2020-01-28 at 10 50 15 PM

**Change List (commits, features, bugs, etc)**

- src/components/AddCard/AddCard.jsx
- src/components/AddCard/_add-card.scss

**Acceptance Test (how to verify the PR)**

- Go to AddCard story and see that the title color matches the icon

### CODE REVIEW

File: src/components/AddCard/AddCard.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const propTypes = {
  */
 const AddCard = ({ onClick, title, className }) => (
   <ClickableTile className={classNames('add-card', className)} handleClick={onClick}>
-    <p className="title">{title}</p>
+    <p className="addcard-title">{title}</p>
```
Comment (davidicus) [inline src/components/AddCard/AddCard.jsx]: We should be using the iot prefix for our unique class names moving forward.

File: src/components/AddCard/__snapshots__/AddCard.story.storyshot [hunk 2]
```diff
@@ -45,7 +45,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT|AddCa
           onKeyDown={[Function]}
         >
           <p
-            className="title"
+            className="addcard-title"
```
Comment (davidicus) [inline src/components/AddCard/__snapshots__/AddCard.story.storyshot]: same comment about prefix

File: src/components/AddCard/_add-card.scss [hunk 3]
```diff
@@ -15,4 +15,8 @@
   svg {
     margin: auto 0 0 auto;
   }
+
+  p.addcard-title {
```
Comment (davidicus) [inline src/components/AddCard/_add-card.scss]: I would like us to not add to the specificity by using the element selectors.

File: src/components/AddCard/AddCard.jsx (hunk 1 above)
Comment (sstone2423) [reply src/components/AddCard/AddCard.jsx]: Fixed

File: src/components/AddCard/__snapshots__/AddCard.story.storyshot (hunk 2 above)
Comment (sstone2423) [reply src/components/AddCard/__snapshots__/AddCard.story.storyshot]: Fixed

File: src/components/AddCard/_add-card.scss (hunk 3 above)
Comment (sstone2423) [reply src/components/AddCard/_add-card.scss]: Removed specificity

**Pull request merged**